packages/ui/src/working-groups/modals/ApplicationDetailsModal/FormDetails.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+
+import {
+  SidePaneColumn,
+  SidePaneLabel,
+  SidePaneRow,
+  SidePaneTable,
+  SidePaneText,
+} from '../../../common/components/SidePane'
+import { useApplicationQuestionAnswers } from '../../hooks/useApplicationQuestionAnswers'
+import { ApplicationFormQuestionAnswer } from '../../types/ApplicationFormQuestionAnswer'
+
+interface Props {
+  applicationId: string
+}
+
+export const FormDetails = React.memo(({ applicationId }: Props) => {
+  const { isLoading, answers } = useApplicationQuestionAnswers(applicationId)
+
+  if (isLoading)
+    return (
+      <SidePaneTable>
+        <SidePaneText>Loading...</SidePaneText>
```

Comment:
Please use `<Loading />` component. There a chance that we add something fancy for loaders and then we will have one place to change :)